### PR TITLE
kebak-case

### PR DESCRIPTION
**Pull request by JcMinarro**
Use kebab-case for the id of every file instead of camelCase

### CODE REVIEW

File: src/test/kotlin/io/getstream/dokkasaurus/utils/StringExtensionsTest.kt [hunk 1]
```diff
@@ -21,4 +24,25 @@ internal class StringExtensionsTest {
...
+    companion object {
+        @JvmStatic
+        fun generateCamelToSnakeCaseArguments() = listOf(
+            Arguments.of("a", "a"),
+            Arguments.of("A", "a"),
+            Arguments.of("ab", "ab"),
+            Arguments.of("aB", "a-b"),
+            Arguments.of("AB", "a-b"),
+            Arguments.of("ABc", "a-bc"),
+            Arguments.of("ABC", "a-b-c"),
```
Comment (zsmb13) [inline src/test/kotlin/io/getstream/dokkasaurus/utils/StringExtensionsTest.kt:43]: This is kebab-case 😄  
image

Comment (JcMinarro) [reply src/test/kotlin/io/getstream/dokkasaurus/utils/StringExtensionsTest.kt:43]: UPs, yep

File: src/test/kotlin/io/getstream/dokkasaurus/utils/StringExtensionsTest.kt [hunk 2]
```diff
@@ -21,4 +24,25 @@ internal class StringExtensionsTest {
...
+
+    @ParameterizedTest
+    @MethodSource("generateCamelToSnakeCaseArguments")
+    fun `String should be converted from camelCase to kebab-case`(input: String, expectedResult: String) {
+        Assertions.assertEquals(expectedResult, input.camelToKebabCase())
+    }
+
+    companion object {
+        @JvmStatic
+        fun generateCamelToSnakeCaseArguments() = listOf(
```
Comment (leandroBorgesFerreira) [inline src/test/kotlin/io/getstream/dokkasaurus/utils/StringExtensionsTest.kt:36]: `generateCamelToKebabCaseArguments` , no?

File: src/main/kotlin/io/getstream/dokkasaurus/utils/StringExtensions.kt [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
 
 internal fun String.simpleScapeTags(): String {
     return this.replace("<", "&lt;").replace(">", "&gt;")
 }
+
+internal fun String.camelToKebabCase(): String {
+    return camelRegex.replace(this) {
+        "-${it.value}"
+    }.toLowerCase()
+}
```
Comment (leandroBorgesFerreira) [inline src/main/kotlin/io/getstream/dokkasaurus/utils/StringExtensions.kt:12]: line break

**Review by leandroBorgesFerreira — APPROVED** [review]
Just 2 comments

**Pull request merged**